pkg/udev/common.go

```diff
@@ -107,7 +107,10 @@ func (device *UdevDevice) GetUid() string {
 	// 	On Gke, we have the ID_TYPE property, but still disks will have
 	// same attributes. We have to put a special check to handle it and process
 	// it like a Virtual disk.
-	if len(idtype) == 0 || model == "EphemeralDisk" {
+	localDiskModels := make([]string, 0)
+	localDiskModels = append(localDiskModels, "EphemeralDisk")
```

Comment:
@kmova,  now I am afraid if there are more cases like this. If customer is trying with new type of EphemeralDisk disk then we can not help him without making any code changes. Hardcoding it in the code will not scale.
 Can we pass this info from env which we can set in the yaml. We can have VDISK_MODEL_LIST environment, and if it is not set we will use EphemeralDisk and Virtual_disk by default?